WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1631,6 +1634,13 @@ static GSourceFuncs _handlerIntervention =
             g_free(wpeStoragePath);
             WKContextConfigurationSetLocalStorageDirectory(contextConfiguration, storageDirectory);
 
+            if (_config.LocalStorageSize.IsSet() == true && _config.LocalStorageSize.Value().empty() == false) {
+                uint32_t val = std::stoul(_config.LocalStorageSize.Value().c_str());
+                uint32_t gLocalStorageDatabaseQuotaInBytes = val * 1024;
+                TRACE_L1("Configured LocalStorage Quota  %u bytes", gLocalStorageDatabaseQuotaInBytes);
```

Comment:
Although I know there are still more places where TRACE_L1 is used in this file that is deprecated and it would be better to use the TRACE(Trace::Information, construction (also already used here). That way you would have the traces also available in a release build and configurable.